code/modules/awaymissions/maploader/reader.dm

```diff
@@ -321,7 +325,7 @@ var/global/dmm_suite/preloader/_preloader = null
 /dmm_suite/preloader/New(list/the_attributes, path)
 	.=..()
 	if(!the_attributes.len)
-		Del()
+		qdel(src)

```

Comment:
Fair enough--I generally leave thing with the dmm_suit alone because of how ancient it is.